applications/tari_base_node/src/parser.rs

```diff
@@ -675,7 +675,7 @@ impl Parser {
 
             let mut height = meta.height_of_longest_chain.expect("Could not retrieve chain height");
 
-            let mut current_header = node
+            let current_header = node
```

Comment:
you can remove this.
I think this stuck around after a refactor

Reply:
Will remove in next PR.